docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+web:
+  build: .
+  ports:
+    - "8888:8888"
```

Comment:
I don't think this is necessary because we're not opening this port to any of the other containers and already exposing port 8888... not 100% sure tho.